### PR TITLE
pass correct tracker store object #1277

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- https://github.com/RasaHQ/rasa_core/issues/1277
- If no queue is present for the broker configuration, it overwrites the default queue with `None`. This should not be the case, the default value should be preserverd

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] updated the changelog

### CODE REVIEW

**Comment by wochinge** [comment]
@tmbo 
- do we maintain`rasa_core/server.py`? Since at the moment `server.py` does not implement the new tracker store + broker feature
- do we need an update in the changelog

**Comment by wochinge** [comment]
@tmbo updated

File: rasa_core/server.py [hunk 1]
```diff
@@ -600,38 +598,5 @@ def tracker_predict():
...
-    run.serve_application(_agent,
-                          cmdline_args.connector,
-                          cmdline_args.port,
-                          cmdline_args.credentials,
-                          cmdline_args.cors,
-                          cmdline_args.auth_token,
-                          cmdline_args.enable_api,
-                          cmdline_args.jwt_secret,
-                          cmdline_args.jwt_method)
+    raise RuntimeError("Using `rasa_core.server` is not longer supported. "
```
Comment (tmbo) [inline rasa_core/server.py]:
```suggestion
    raise RuntimeError("Calling `rasa_core.server` directly is no longer supported. "
```

**Comment by tmbo** [comment]
:D the build failed because of pep8 & the line I added being to long - very unfortunate that githubs code suggestion feature doesn't show line length :wink:

**Comment by wochinge** [comment]
Yeah, I had this one before 😄 Fix is pushed.

**Pull request merged**